sky/serve/service_spec.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         readiness_timeout_seconds: int,
         min_replicas: int,
         max_replicas: Optional[int] = None,
+        port: Optional[int] = None,
```

Comment:
```suggestion
        ports: Optional[str] = None,
```
change name to `ports` for future compatibiility?